packages/api/src/dataFeeds.json

```
@@ -9,7 +9,9 @@
     "witnetRequestBoard": {
       "address": "0x400DbF3645b345823124aaB22D04013A46D9ceD5",
       "abi": "./src/abi/WitnetRequestBoardProxy.json"
-    }
+    },
+    "color": "#41BEA562",
+    "blockExplorer": "https://etherscan.io/address/{address}#code"
```

Can we drop the `#code` part in each etherscan link? @mariocao noticed this recently
```suggestion
    "blockExplorer": "https://etherscan.io/address/{address}"
```